lib/network/rest/RESTServer.js

```diff
@@ -73,6 +73,12 @@ function parseURL(urlStr, expectKey) {
     return pathInfo;
 }
 
+// This is set to a higher value than rest client's free socket
+// timeout (55 seconds) to avoid the risk of triggering ECONNRESET
+// errors if the server closes the connection at the time rest client
+// attempts to reuse it.
+const KEEP_ALIVE_TIMEOUT = 60000;
```

Comment:
should we reuse https://github.com/scality/Arsenal/blob/development/7.4/lib/constants.js#L85 ?

Reply:
done